src/formatDate.js

```diff
@@ -9,6 +9,27 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const data = {};
+  const newDateFormat = [];
+  const dateParts = date.split(fromFormat.slice(3));
```

Comment:
The `split` method is not used correctly. The `fromFormat.slice(3)` will return an array, and `split` expects a string or a regular expression. You should use the last element of `fromFormat` as the separator.